cypress/integration/ViewTermsByLetter.feature

```diff
@@ -0,0 +1,25 @@
+Feature: View Terms By Letter
+
+	Scenario: User appends /expand to the URL and tries to access the dictionary
+		Given the user navigates to "/expand"
+		Then the system returns the no results found page
+
+	Scenario: User selects a letter from A-Z list on dictionaries for terms in lists
+		Given user is on the dictionary landing page or results page
+		When user selects letter "A" from A-Z list
+		Then the system returns user to the search results page for the search term "A" URL has "/expand"
+		Then search results page displays results title "# results found for: A"
+		And each result in the results listing appears as a link to the term's page
+		# And each result that is an other name that is displayed in results is displayed with its Other Name, with (Other name for: [preferred name]) as its display
+		And each result displays its full definition below the link for the term
+
+	Scenario: Results /expand page metadata
+		Given the user is viewing a results page based on clicking a letter like "A" in the dictionary
```

Comment:
same here - why not just use "Given the user navigates to "expand/a"? - this step only does navigation, no need to add multiple steps to navigate to the url ( I know we did it for the glossary, but for the sake of simplifying the tests lets just use 'user navigates to' for all navigation steps)

Reply:
👍